lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -291,6 +294,11 @@ def add_test_targets
               create_test_target_embed_frameworks_script(test_type)
               create_test_target_copy_resources_script(test_type)
 
+              # Generate vanila Info.plist for test target similar to the one xcode gererates for new test target.
+              # This creates valid test bundle accessible at the runtime, allowing tests to load bundle resouces.
+              test_info_plist_path = project.path.dirname + "#{name}/#{test_type.capitalize}TestInfo.plist"
```

Comment:
this is a good fix, does it happen for normal targets within their create target method or is it a separate method?

Reply:
This path seems wrong 
![screen shot 2018-03-19 at 5 33 40 pm](https://user-images.githubusercontent.com/310370/37629372-b6277ff6-2b9b-11e8-8454-3bc37f8b26eb.png)

The file reference is actually OK because its a different path but it should go under `Target Support Files` to be organized like the rest of them.

Reply:
got it